src/cache.h

```diff
@@ -7,6 +7,8 @@
 
 typedef struct CacheQuery
 {
+	/* Set to only find the cache entry (do not create) */
+	bool		findonly;
```

Comment:
I don't see where this is used. cache.c doesn't seem to have any changes anymore.

Reply:
No it is not used, but I left it in there because I think it might be useful for the future. For instance, when we do cache invalidation on a relid, we can check whether that key is in the cache, if not, we need not invalidate the cache.

Reply:
Let's remove this. We have a long-standing policy of not leaving unused stuff in. We can put back later.